setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name='pulp-rpm',
-    version='0.0.1a1.dev1',
+    version='3.0.0b1',
```

Comment:
Is it 3.0? I can see value in that because then everyone knows. I had imagined it would be 0.1.0b1 personally. What should we do?